doc/man/8/mount.ceph.rst

```diff
@@ -61,9 +61,10 @@ Basic
     for autodiscovery of monitor addresses and auth secrets. The default is
     to use the standard search path for ceph.conf files.
 
-:command:`mds_namespace=<fs-name>`
+:command: `fs=<fs-name>`
     Specify the non-default file system to be mounted. Not passing this
-    option mounts the default file system.
+    option mounts the default file system. Or the deprecated ``mds_namespace=<fs-name>``
+    for backward compatibility.
```

Comment:
I'd add a new ":command:" stanza in here to cover the legacy "mds_namespace=" option. You can just say that it's a synonym of "fs=" and that its use is deprecated.

Reply:
Will fix it.